projects/kc63_preprocessing/reporting_period/add_reporting_period.py

```diff
@@ -0,0 +1,48 @@
+import re
+import pandas as pd
+from pathlib import Path
+from typing import Iterable
+
+YEAR_SUFFIX_RE = re.compile(r"(\d{4})$")
+
+
+def extract_year_from_filename(filename: str) -> int:
+    stem = Path(filename).stem
+    m = YEAR_SUFFIX_RE.search(stem)
```

Comment:
Nit-pick: clarity; does `m` mean `match`?

```suggestion
    match = YEAR_SUFFIX_RE.search(stem)
```

Reply:
Updated